content/docs/user-guide/experiment-management/experiments.md

```diff
@@ -0,0 +1,33 @@
+## Experiments
+
+_New in DVC 2.0_
+
+`dvc exp` commands let you automatically track a variation to an established
+[data pipeline](/doc/command-reference/dag). You can create multiple isolated
+experiments this way, as well as review, compare, and restore them later, or
+roll back to the baseline. The basic workflow goes like this:
+
+- Modify stage <abbr>parameters</abbr> or other dependencies (e.g. input data,
```

Comment:
not for this PR: Modify hyperparameters (link to params), code ... etc ... we are being way too formal here again. I thinks we can sacrifice it a bit and use some common ML terminology